### ISSUE TITLE
other backends?

### ISSUE THREAD

**Issue by Mistress-Anna** [PhotoBackup/client-android#6]
Maybe photos (and videos in the future?) could be uploaded to an owncloud host or an sftp server?

**Comment by stephanepechard** [comment]
Yes, other backends are scheduled (in my mind for the least, not specifically in time).
SFTP would probably be one of the first to be done.
